terraform/gcp/security/gcp-org-impersonation-roles-iam-member.yaml

```diff
@@ -0,0 +1,23 @@
+rules:
+    - id: gcp-org-impersonation-roles-iam-member
+      patterns:
+        - pattern: resource
+        - pattern-inside: |
+              resource "google_organization_iam_member" "..." {
+              ...
+              role = "roles/editor"
+              member = ...

```

Comment:
Same question as bove

Reply:
Yes, the issue is with the editor role (https://cloud.google.com/iam/docs/understanding-roles).